### PR TITLE
Closes #362: Correct layout related things on main dataverse page

### LINKED ISSUE #362: Correct layout related things on main dataverse page
- distance between logo and dataverse name on carousel
- when single option in dataverse/dataset/datafile then it should be disabled
- align harvested/linked icon
- align of file tags

### PR DESCRIPTION
**Pull request by sudmar**
- increased distance between dataverse logo and name on the carousel
- checkboxes are now greyed out when disabled
- fixed the alignment of the harvested/linked icons
- fixed the alignment of the dataverse icons (replaced with an SVG image)
- fixed the look of the error messages

### CODE REVIEW

File: dataverse-webapp/src/main/webapp/resources/vecler/theme.scss [hunk 1]
```diff
@@ -21,6 +21,7 @@ $search-result-desc-color: #f2f8fa;
 $table-row-hover: #e6eff2;
```
Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:21]: Harvested and linked icons on file are grey
harvested

Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:21]: I have a feeling that linked/harvested icons are bigger than dataset/file icons.

**Review by madryk — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/webapp/resources/vecler/theme.scss (hunk 1 above)
Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:21]: Fixed

Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:21]: Made them a bit smaller

**Review by madryk — APPROVED** [review]

**Pull request merged**